controls/nist_rhcos4.yml

```diff
@@ -371,8 +371,13 @@ controls:
   - high
   - moderate
 - id: AC-2(11)
-  status: pending
+  status: not applicable
   notes: |-
+    Enforcement of organization-defined circumstances and/or usage
+    conditions is the responsibility of third party identity
+    management software such as Red Hat Identity Management (Red Hat IdM),
+    Active Directory (MSAD), LDAP, etc.
+
```

Comment:
I think your addition is good, but can you remove the text right under it that says that engineering response is planned?

Reply:
Sure thing